README.md

```diff
@@ -2,13 +2,17 @@
 
 A book uploader bot that transfers documents from public libraries such as Google Books, and Panjab Digital Library etc to Internet Archive. Built for Wikimedia Tool Labs. Check out [BUB2 on Toolforge](https://bub2.toolforge.org)!
 
-## Getting Started
+![](https://img.shields.io/github/repo-size/coderwassananmol/BUB) ![](https://img.shields.io/github/license/coderwassananmol/BUB2?color=red)<br>
```

Comment:
Can you move the fork/stars/watch badges to right of the BUB2 heading?
And rest of the badges just below the heading? Above "A book uploader bot ..."